src/pages/[event]/index.tsx

```diff
@@ -36,6 +36,7 @@ export default function Event({
     year,
   },
 }: ReturnedProps<typeof getStaticProps>) {
+  const eventColors = EventColors({ event: slug });
```

Comment:
[Suggestion] I would move this into `FindUs`. The `waterColor` value isn't used until the `onLoad` callback inside that component, and the component also receives the `slug`. Fewer lines of code for the same result.

Reply:
Good call!